red-pill-contracts-py/tests/test_agent.py

```diff
@@ -0,0 +1,31 @@
+"""Test the agent on a fork"""
```

Comment:
you mentioned on chat these tests are no good? should we remove those?

Reply:
i updated them, i just miss an actual transaction log of a red-pill contract